src/Common/src/Interop/Windows/NtDll/Interop.NtCreateFile.cs

```diff
@@ -4,13 +4,15 @@
 
 using System;
 using System.Runtime.InteropServices;
+using System.Data.Common;
 
 internal partial class Interop
 {
     internal partial class NtDll
     {
         // https://msdn.microsoft.com/en-us/library/bb432380.aspx
         // https://msdn.microsoft.com/en-us/library/windows/hardware/ff566424.aspx
+#pragma warning disable BCL0015 // Invalid Pinvoke call
```

Comment:
Suppressing warnings as per Jan's suggestion #29169

Reply:
Please remove the pragma- shouldn't need it if we aren't including this on UAP.

---
In reply to: [190421828](https://github.com/dotnet/corefx/pull/29879#discussion_r190421828) [](ancestors = 190421828)